packages/terra-layout/src/HoverTarget.scss

```diff
@@ -0,0 +1,85 @@
+:local {
+  .hover-target {
+    overflow: hidden;
+    width: 100%;
+  }
+
+  .content-section {
+    float: left;
+    -webkit-overflow-scrolling: touch;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+    overflow-y: auto;
+    width: calc(100% - var(--layout-menu-hover-target-width, 30px));
+  }
+
+  .hover-section {
+    align-items: stretch;
+    background: var(--layout-menu-hover-target-background-color, #e8e9ea);
+    border: 0;
+    border-right: var(--layout-menu-hover-target-border-right, 1px solid #bcbfc0);
+    display: flex;
+    flex-direction: column;
+    float: right;
+    overflow: hidden;
+    width: var(--layout-menu-hover-target-width, 30px);
+
+    &:active {
+      background: var(--layout-menu-hover-target-background-color, #c6c7c9);
+    }
+  }
+
+  .hover-section-icon {
+    flex: 0 0 auto;
+    padding-top: 10px;
```

Comment:
@bjankord was mentioning px values should be updated to rem on my Field PR.

Reply:
Yeah UX is big on using rem for layout, I agree. From an accessibility perspective, we should be using rems. This can be updated to `0.7143rem`